state-sync/storage-service/server/src/metrics.rs

```diff
@@ -127,12 +130,32 @@ pub static STORAGE_RESPONSES_SENT: Lazy<IntCounterVec> = Lazy::new(|| {
     .unwrap()
 });
 
-/// Time it takes to fetch a storage service response
+/// Time it takes to read data from the storage service DB
+pub static STORAGE_DB_READ_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
+    register_histogram_vec!(
+        "aptos_storage_service_server_db_read_latency",
+        "Time it takes to read data from the storage service DB",
+        &["request_type", "result"],
+    )
+    .unwrap()
+});
```

Comment:
bucket?

Reply:
Do you mean add custom buckets to these latencies? The defaults should already be good enough, no? i.e., none of these should go beyond 10 seconds, and I don't need that fine a granularity 😄 Or did you mean something else?